packages/core/src/credentials/index.ts

```diff
@@ -0,0 +1,19 @@
+import { parseKey } from '@preevy/common'
+import { memoize } from 'lodash'
+
+export const getUserCredentials = memoize((privateKey: string | Buffer) => {
+  const parsedKey = parseKey(privateKey)
+  const user = 'x-preevy-profile-key'
```

Comment:
User can be replaced in favor of data that contains information about the password/token scope. (like timestamp, or jwt structure)